languages/csharp/exercises/concept/tuples/Tuples.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+public class PhoneNumbers
```

Comment:
```suggestion
public static class PhoneNumbers
```

Reply:
Tiny nit: I'm used to using singular when possible, so `PhoneNumber` instead of `PhoneNumber`. To me `PhoneNumber.Analyzer` and `PhoneNumber.IsFake` reads slightly better than `PhoneNumbers.Analyzer` and `PhoneNumbers.IsFake`. So maybe do:

```suggestion
public static class PhoneNumber
```

If you like this, the tests and example file also need to be updated.